src/ape_ethereum/transactions.py

```diff
@@ -461,10 +461,26 @@ def _decode_ds_note(self, log: Dict) -> Optional[ContractLog]:
 
 
 class SharedBlobReceipt(Receipt):
+    """
+    An `EIP-4844 <https://eips.ethereum.org/EIPS/eip-4844#blob-transaction>`__"
+    blob transaction.
+    """
+
     blob_gas_used: int
```

Comment:
alternatively i could make one or both of these optional.
i really dont know, ive been reading through the EIP but just really know